### PR TITLE
843754 Rich Text Editor provides support to IsScrollButtonVisible feature

### PR DESCRIPTION
**Pull request by HemalathaDuraisamy**
Rich Text Editor provides support to IsScrollButtonVisible feature

### CODE REVIEW

File: Xamarin/Rich-Text-Editor/Customization.md [hunk 1]
```diff
@@ -83,6 +83,29 @@ this.Content = richtexteditor;
 
 {% endtabs %}
 
+### Scroll button support on toolbar
+
+The Rich Text Editor provides support to scroll the benind the Toolbar visible items .The IsScrollButtonVisible Property help to enable and disable the scroll on Toolbar. The default value is false.The following code example explains how to customize the toolbar appearance.
```
Comment (M-A32) [inline Xamarin/Rich-Text-Editor/Customization.md:88]: The Rich Text Editor provides support to scroll the benind the Toolbar visible items .The IsScrollButtonVisible Property help to enable and disable the scroll on the Toolbar. The default value is false. The following code example explains how to customize the toolbar appearance.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: Xamarin/Rich-Text-Editor/Customization.md [hunk 2]
```diff
@@ -83,6 +83,29 @@ this.Content = richtexteditor;
 
 {% endtabs %}
 
+### Scroll button support on toolbar
+
+The Rich Text Editor provides support to scroll the benind the Toolbar visible items .The IsScrollButtonVisible Property help to enable and disable the scroll button on the Toolbar. The default value of IsScrollButtonVisible is false.The following code example explains how to customize the toolbar appearance.
```
Comment (Mohanaraja) [inline Xamarin/Rich-Text-Editor/Customization.md:88]: The Rich Text Editor provides support to scroll behind the Toolbar visible items. The IsScrollButtonVisible Property helps to enable and disable the scroll button on the Toolbar. The default value of IsScrollButtonVisible is false. The following code example explains how to customize the toolbar appearance.

**Review by Mohanaraja — CHANGES_REQUESTED** [review]
Address the comments

File: Xamarin/Rich-Text-Editor/Customization.md [hunk 3]
```diff
@@ -83,6 +83,29 @@ this.Content = richtexteditor;
 
 {% endtabs %}
 
+### Scroll button support on toolbar
+
+The Rich Text Editor provides support to scroll behind the Toolbar visible items. The IsScrollButtonVisible Property helps to enable and disable the scroll button on the Toolbar. The default value of IsScrollButtonVisible is false. The following code example explains how to customize the toolbar appearance.
+
```
Comment (Mohanaraja) [inline Xamarin/Rich-Text-Editor/Customization.md:89]: Use the below content 

The Rich Text Editor allows scrolling behind the toolbar's visible items. You can enable or disable the scroll button on the toolbar using the IsScrollButtonVisible Property. By default, the value of IsScrollButtonVisible is set to false. The code example below shows how to customize the appearance of the toolbar.

**Review by Mohanaraja — CHANGES_REQUESTED** [review]
Address the comments

File: Xamarin/Rich-Text-Editor/Customization.md (hunk 3 above)
Comment (HemalathaDuraisamy) [reply Xamarin/Rich-Text-Editor/Customization.md:89]: Updated the mentioned corrections

**Review by Mohanaraja — APPROVED** [review]
changes are fine

**Pull request merged**